ui/summit-2024/src/components/AppWrapper/AppWrapper.tsx

```diff
@@ -144,10 +145,14 @@ const AppWrapper = (props: { children: ReactNode }) => {
       dispatch(setEventCache(eventDataFixture));
     } else {
       const eventData = await getEventData(env.EVENT_ID);
+      const eventDataExtended =
+        env.TARGET_NETWORK == "MAINNET"
```

Comment:
You can use `NetworkType.MAINNET` instead of `"MAINNET"`

Reply:
True. Thank you!